ios/FluentUI.Demo/FluentUI.Demo/Demos/TableViewCellFileAccessoryViewDemoController.swift

```diff
@@ -519,8 +519,8 @@ class TableViewCellFileAccessoryViewDemoController: DemoTableViewController {
 
         for accessoryView in topAccessoryViews + bottomAccessoryViews {
             if let cell = accessoryView.tableViewCell {
-                cell.paddingLeading = TableViewCell.defaultPaddingLeading + extraPadding
-                cell.paddingTrailing = TableViewCell.defaultPaddingTrailing + extraPadding
+                cell.paddingLeading = TableViewCell().defaultPaddingLeading + extraPadding
+                cell.paddingTrailing = TableViewCell().defaultPaddingTrailing + extraPadding
```

Comment:
Couldn't these defaults be retrieved by just creating a tokens class a returning the properties in it? (since it's not being added to a window and will use the default implementation)

Reply:
Yeah, I would prefer we avoid creating throwaway instances of any view whenever possible (especially two throwaway instances like here!).